(SIMP-3291)  processgraph: Configure Travis to publish gem and release

lnemsick-simp opened the pull request:
When the repo is tagged, publish to rubygems.org and release on github.

NOTE:  At this time, the travis build is NOT expected to pass.

lnemsick-simp committed:
(SIMP-3291)  Configure Travis to publish gem and release

When the repo is tagged, publish to rubygems.org and release on github.

nick-markowski reviewed (dismissed):
Although the ticket is to configure travis for push, I think we need to give this a little bit of TLC so we can get tests tooled to pass in travis

lnemsick-simp committed:
Pin to ruby 2.0.0 and fix Ubuntu packages

On CentOS, we have to use ruby 2.0.0, so try to pin to ruby 2.0.0 here.  May or may not work with our guess at Ubuntu packages.

judyj committed:
Update .travis.yml

changed rvm to use system version of ruby

judyj committed:
Update .travis.yml

install bundle manually because we are not using RVM

judyj committed:
Update Gemfile

We do not need the gv gem - it confuses the library

judyj committed:
Update simp-processgraph.gemspec

Commented out s.metadata -- does not seem to work in ruby < 2, so it fails the travis test unless we have it commented out

judyj committed:
Update .travis.yml

lnemsick-simp committed:
Update .travis.yml

Commit:
Update Rakefile
